core_impl/src/test/java/io/opencensus/tags/UnreleasedApiAccessor.java

```diff
@@ -11,9 +11,23 @@
  * limitations under the License.
  */
 
-package io.opencensus.impl.tags;
+package io.opencensus.tags;
 
-import io.opencensus.tags.TagContextFactory;
+import io.opencensus.tags.TagKey.TagKeyBoolean;
+import io.opencensus.tags.TagKey.TagKeyLong;
 
-final class TagContextFactoryImpl extends TagContextFactory {
+/**
+ * Class for accessing methods in the {@code io.opencensus.tags} package that haven't been made
+ * public yet.
+ */
+public final class UnreleasedApiAccessor {
```

Comment:
This class needs to be in the same package as the tags API to access the non-public methods.  Also, I don't think it needs a warning, since it is just a test class and it won't be used outside of this project.